DuckDuckGo/AppDelegate.swift

```diff
@@ -192,23 +192,12 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         // MARK: Sync initialisation
 
         syncDataProviders = SyncDataProviders(bookmarksDatabase: bookmarksDatabase, secureVaultErrorReporter: SecureVaultErrorReporter.shared)
-        syncService = DDGSync(dataProvidersSource: syncDataProviders, errorEvents: SyncErrorHandler(), log: .syncLog)
+        let syncService = DDGSync(dataProvidersSource: syncDataProviders, errorEvents: SyncErrorHandler(), log: .syncLog)
         syncService.initializeIfNeeded(isInternalUser: InternalUserStore().isInternalUser)
-        syncStateCancellable = syncService.authStatePublisher
-            .prepend(syncService.authState)
-            .map { $0 == .inactive }
-            .removeDuplicates()
-            .sink { [weak self] isSyncDisabled in
-                self?.syncDataProviders.credentialsAdapter.updateDatabaseCleanupSchedule(shouldEnable: isSyncDisabled)
-                self?.syncDataProviders.bookmarksAdapter.updateDatabaseCleanupSchedule(shouldEnable: isSyncDisabled)
-            }
-        syncDataProviders.bookmarksAdapter.databaseCleaner.isSyncActive = { [weak self] in
-            self?.syncService.authState == .active
-        }
-        syncDataProviders.credentialsAdapter.databaseCleaner.isSyncActive = { [weak self] in
-            self?.syncService.authState == .active
+        if application.applicationState == .active {
```

Comment:
I think e can remove this - isn't the app state here always `inactive` ?

Reply:
I think it can transition to `background` or `active` depending on the way app has been launched, but at this point I'd expect this to be always `inactive`.

Reply:
Of course, your right. Cool, let me remove it.